src/server/controllers/VerificationController.ts

```diff
@@ -952,6 +955,16 @@ export default class VerificationController
         .custom(
           (chain, { req }) => (req.chain = this.validateSingleChainId(chain))
         ),
+      body("msgSender")
```

Comment:
isn't it better to handle the the global variables as an object in which msgSender is one of the properties?